org.jousse.vincent.Pomodorolm.yml

```diff
@@ -0,0 +1,84 @@
+id: org.jousse.vincent.Pomodorolm
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.node20
+
+command: pomodorolm
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --socket=pulseaudio # We want to be able to play sounds
+  - --device=dri # OpenGL
+  - --share=ipc
+  - --env=FLATPAK=1 # Used to tell the app to look for assets in /app/lib/pomodorolm
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+build-options:
+  append-path: /usr/lib/sdk/node20/bin:/usr/lib/sdk/rust-stable/bin
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: pomodorolm
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/vjousse/pomodorolm.git
+        tag: app-v0.1.10
+        commit: 73597e8f09120588fd2b5cc0f3c0d3e22e3eb306
+
+      # flatpak-node-generator npm -o flatpak/node-sources.json package-lock.json
+      - node-sources.json
+
+      # flatpak-cargo-generator.py -d ./src-tauri/Cargo.lock -o flatpak/cargo-sources.json
+      - cargo-sources.json
+
+      # flatpak-elm-generator.py ../elm.json elm-sources.json
+      - elm-sources.json
+
+      - type: file
+        path: shelm
```

Comment:
Let's use this as URL

```
- type: file
  url: https://raw.githubusercontent.com/robx/shelm/4417730f5847e6ccba1b19f1b25166471433d633/shelm
  sha256:  ....
```

Copying it may cause copyright issues.

Reply:
Should be fixed (with an additional patch file)